features/steps/kafka_steps.py

```diff
@@ -41,25 +41,30 @@ def retrieve_broker_metadata(context, hostname=None, port=None):
     out = subprocess.Popen(
         ["kcat", "-b", address, "-L", "-J"],
         stdout=subprocess.PIPE,
-        stderr=subprocess.STDOUT,
+        stderr=subprocess.PIPE,
     )
 
     stdout_file = filepath_from_context(context, "logs/insights-results-aggregator/", "_stdout")
+    stderr_file = filepath_from_context(context, "logs/insights-results-aggregator/", "_stderr")
 
     # interact with the process:
     # read data from stdout and stderr, until end-of-file is reached
     stdout, stderr = out.communicate()
 
-    assert stderr is None, "Error during check"
     assert stdout is not None, "No output from process"
```

Comment:
If the output is expected in stderr, can it be now that stdout is empty?

Reply:
No, as both outputs are redirected to a "pipe", both variables should be valid, even if they are empty.

Now the output won't be expected in stderr: this change makes that both output are separated and treated different. Previously, the stderr was redirected to the stdout, so all the messages were received in the same `stdout` variable, making `stderr` to contain a `None`